Makefile

```diff
@@ -81,6 +84,10 @@ $(pokecrystal11_obj):       RGBASMFLAGS += -D _CRYSTAL11
 $(pokecrystal_au_obj):      RGBASMFLAGS += -D _CRYSTAL11 -D _CRYSTAL_AU
 $(pokecrystal_debug_obj):   RGBASMFLAGS += -D _DEBUG
 $(pokecrystal11_debug_obj): RGBASMFLAGS += -D _CRYSTAL11 -D _DEBUG
+$(pokecrystalvc_obj):       RGBASMFLAGS += -D _CRYSTAL -D _CRYSTAL11 -D _CRYSTALVC
+
+%.patch: %.patch.template %.gbc pokecrystal11.gbc
```

Comment:
Ideally the patch shouldn't specifically depend on pokecrystal11.gbc; that's only there because of how it currently calculates patch sizes, right?

Reply:
Does the `$(pokecrystalvc_obj)` line really need the `-D _CRYSTAL`  Shouldn't it really only be `-D _CRYSTAL11 -D _CRYSTALVC` in the current state of the PR? This was how it was in @mid-kid code originally I believe.

Reply:
@vulcandth The `-D _CRYSTAL` can be removed. Then all that should be left is refactoring `make_patch.c`, including a new way of dealing with patch contents that's not the current "`big`" approach with diffing ROMs.

Reply:
Removed `-D _CRYSTAL` in https://github.com/pret/pokecrystal/pull/882/commits/bab813661aa857f253ec8d5ee159379346739271. The make_patch.c will definitely need a bit of rewriting.